File: cpp/exercicios cpp/iniciante/bee1015.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

/*
Leia os quatro valores correspondentes aos eixos x e y de dois pontos quaisquer no plano, p1(x1,y1) e p2(x2,y2)
e calcule a distância entre eles, mostrando 4 casas decimais, segundo a fórmula:

Distancia = raiz (x2 - x1)² + (y2 - y1)²
*/

// Protótipo de funções
float distancia(float, float, float, float);

// Função principal
int main(){

    // variáveis para calcular 
    float X1, X2, Y1, Y2;

    // entrada de dados do usuário
    scanf("%f", &X1);
    scanf("%f", &Y1);
    scanf("%f", &X2);
    scanf("%f", &Y2);

    // variável para chamada da função
    float raiz = distancia(X1, Y1, X2, Y2);
    float r = sqrt(raiz);

    // impressões 
    printf("%.4f", r);
    return 0;
}

// Funções
float distancia(float x1, float y1, float x2, float y2){
    
    float distancia = ((x2 - x1) * 2) + ((y2 + y1) * 2);
    return distancia;
}

File: iniciante/python/bee1015.py
# Biblioteca para utilizar a função "math" e usar para calcular uma raiz.
import math

r1 = input().split()
r2 = input().split()

x1 = float(r1[0])
y1 = float(r1[1])
x2 = float(r2[0])
y2 = float(r2[1])

#Fórmula da distância das raízes
distancia = ((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1))
raiz = math.sqrt(distancia)

print('{:.4f}'.format(raiz))